src/KeyUnit.cpp

```diff
@@ -60,14 +60,13 @@ void KeyUnit::uninstall(const QString& packageName)
 
 bool KeyUnit::processDataStream(int key, int modifier)
 {
-    bool isAtLeastOneMatch = false;
     for (auto keyObject : mKeyRootNodeList) {
         if (keyObject->match(key, modifier)) {
-            isAtLeastOneMatch = true;
+            return true;
         }
     }
 
-    return isAtLeastOneMatch;
+    return false;
```

Comment:
That is an edge case scenario. Principle of least surprise is when you press a button and don't get 50 keys going off :)